editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.tsx

```diff
@@ -0,0 +1,198 @@
+import TabContext from "@mui/lab/TabContext";
+import TabPanel from "@mui/lab/TabPanel";
+import Box from "@mui/material/Box";
+import Tab from "@mui/material/Tab";
+import Tabs from "@mui/material/Tabs";
+import Typography from "@mui/material/Typography";
+import { SchemaFields } from "@planx/components/shared/Schema/SchemaFields";
+import { Feature } from "geojson";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React, { useEffect, useState } from "react";
+import FullWidthWrapper from "ui/public/FullWidthWrapper";
+
+import Card from "../../shared/Preview/Card";
+import CardHeader from "../../shared/Preview/CardHeader";
+import { MapContainer } from "../../shared/Preview/MapContainer";
+import { PublicProps } from "../../ui";
+import type { MapAndLabel } from "./../model";
+import { MapAndLabelProvider, useMapAndLabelContext } from "./Context";
+
+type Props = PublicProps<MapAndLabel>;
+
+function a11yProps(index: number) {
+  return {
+    id: `vertical-tab-${index}`,
+    "aria-controls": `vertical-tabpanel-${index}`,
+  };
+}
+
+const VerticalFeatureTabs: React.FC<{ features: Feature[] }> = ({
+  features,
+}) => {
+  const { schema, activeIndex, formik } = useMapAndLabelContext();
+  const [activeTab, setActiveTab] = useState<string>(
+    features[features.length - 1].properties?.label || "",
+  );
+
+  const handleChange = (_event: React.SyntheticEvent, newValue: string) => {
+    setActiveTab(newValue);
+  };
```

Comment:
Is `activeTab` actually going to be synonymous with `activeIndex` - do we need to manage both, or just find a way of mapping one onto the other?

Reply:
I think/hope it's just a matter of mapping one onto the other - but TBD a bit ! Trying to find right balance of not having to introduce too many fancy map highlight/zoom-to functionalities right off the bat